.github/actions/spelling/dictionary/apis.txt

```diff
@@ -111,8 +112,9 @@ snprintf
 spsc
 sregex
 STDCPP
-strchr
 STDMETHOD
+strchr
+strcpy
```

Comment:
I just moved `strchr` down because I thought these files were meant to be in alphabetical order, but after looking at the rest of the file more carefully, I'm beginning to think that was more of a suggestion than a rule.

Reply:
I think this ended up conflicting with Leonard's recent touch on this file for the same reason. Alas! Great minds.

Reply:
I had to add quite a bunch of entries to that file and noticed that it wasn't sorted alphabetically.
I'm sorry for the conflicts this causes.

Reply:
No worries. I was just being lazy and ignoring the issue, so I'm glad you cleaned it up.